pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -57,7 +57,7 @@ const (
 	defaultResolveConfPath             = "/etc/resolv.conf"
 	defaultLocalDNS                    = "svc.cluster.local"
 	tigeraElasticsearchUserSecretLabel = "tigera-elasticsearch-user"
-	defaultElasticsearchShards         = 5
+	defaultElasticsearchShards         = 1
```

Comment:
I think we need a separate PR to make this happen for Helm based installations. Maybe it can be dont from calico-private with an env setting, but probably we want to lower the default in the docker-fluentd repo as well.

Reply:
Do we not have independent control of this for each type of index?  I think we do sometimes want to have more shards (probably based on ES resources) for the flow logs index.